extensions/git-ui/src/typings/refs.d.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/// <reference path='../../../../src/vs/vscode.d.ts'/>
+/// <reference path='../../../../src/vs/vscode.proposed.d.ts'/>
+/// <reference types='@types/node'/>
```

Comment:
Is this still needed?

Reply:
It seems so. The Git extension also has its refs.d.ts. 🤷‍♂

Reply:
I wouldn't take the Git extension as an example, it's a dinosaur by now. 🦕